en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -22,21 +22,25 @@ To start multiple instances (on separate ports):
    ```
    ./Tools/sitl_multiple_run.sh 2
    ```
-1. Start the first instance:
+1. Start the first instance in the same terminal (this will run in the foreground):
    ```
    ./Tools/jmavsim_run.sh -l
    ```
-1. Start subsequent instances, specifying the *simulation* TCP port for the instance:
+1. Open a new terminal for each subsequent instance, specifying the *simulation* TCP port for the instance:
    ```
-   ./Tools/jmavsim_run.sh -p 4561 -l
+   ./Tools/jmavsim_run.sh -p 4560 -l
```

Comment:
Note, I think this means it starts at 4560 right? i.e. did you start the first one at 4560 or 4561 ?